src/Misc/layoutroot/run.sh

```diff
@@ -27,9 +27,9 @@ else
     "$DIR"/bin/Runner.Listener run $*
 
 # Return code 4 means the run once runner received an update message.
```

Comment:
update the comment as well?
` Return code 3 means the run once runner received an update message.`